src/core/unit.c

```diff
@@ -6626,6 +6628,41 @@ static uint64_t unit_get_cpu_weight(Unit *u) {
         return cc ? cgroup_context_cpu_weight(cc, manager_state(u->manager)) : CGROUP_WEIGHT_DEFAULT;
 }
 
+void unit_get_exec_quota_stats(Unit *u, ExecContext *c, ExecDirectoryType dt, uint64_t *usage, uint64_t *limit) {
```

Comment:
return params should be named `ret_xyz` in our case. i.e. rename the last two args to `ret_usage` and `ret_limit`